skipper.yaml

```diff
@@ -0,0 +1,14 @@
+registry: quay.io
+build-container-image: special-resource-operator-build
+
+containers:
+  special-resource-operator-build: Dockerfile.special-resource-operator-build
+volumes:
+    - $HOME/.cache/go-build:$HOME/.cache/go-build
+    - $HOME/go/pkg:/go/pkg
+    - $HOME/.cache/golangci-lint:$HOME/.cache/golangci-lint
+    - $HOME/.docker:$HOME/.docker
```

Comment:
I believe this is related to the Docker daemon socket? what happens if the machine doesn't contain `docker` but `podman` ?

Reply:
it should work as is. Docker needs socket since there is a docker daemon running. With podman this mapping will be unused

Reply:
The be accurate this isn't related to the docker daemon socket.
This volume is for mounting the auth file with container registries credentials (usually kept under `$HOME/.docker/config.json`).
It is useful and works for podman as well (though in podman the default path for the auth file is `$XDG_RUNTIME_DIR/containers/auth.json.`)